students/pedro-matas/practica1.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>practica1</title>
+</head>
+<body>
+    <h1 id="inicio">dia triste en la grieta</h1>
+    <img src="https://static1-es.millenium.gg/articles/0/32/09/0/@/148658-derrota-lol-article_image_t-1.jpg" />
+
+    <h2>se viene tarde de derrotas</h2>
+    <p>me han <b>inteado la <i>botlane</i> <u>da que pensar</u> </b></p>
+
+    <ul>
+        <li><a href="https://www.google.es/">link a google</a></li>
+        <li><a href="pedro.matas.ortega@gmail.com">mi correo</li>
+        <li><a href="tel:+34689712180">mi telefono</a></li>
+
+    </ul>
+
+    <form method="post">
+        <ul>
+         <li>
+           <label for="name">Name:</label>
+           <input type="text" id="name" name="user_name">
+         </li>
+         <li>
+            <label for="password">password:</label>
+            <input type="password" name="contraseña" id="password" required>
+          </li>
+         <li>
+           <label for="mail" >E-mail:</label>
+           <input type="email" id="mail" name="user_email" required >
+         </li>
+         <li>
+           <label for="date">date:</label>
+           <input type="date" name="fecha" id="fecha">
+         </li>
+         <li>
+            <label for="chgenero">genero:</label>
+            <input type="checkbox" id="masculino" name="chgenero" value="masculino">
+    <label for="masculino">masculino</label>
+    <input type="checkbox" id="femenino" name="chgenero" value="femenino">
+    <label for="femenino">femenino</label>
+    <input type="checkbox" id="otro" name="chgenero" value="otro">
+    <label for="otro">otro</label>
+          </li>
+          <li>
+            <label for="ragenero">genero</label>
+            <input type="radio" id="masculino" name="chgenero" value="masculino">
+    <label for="masculino">masculino</label>
+    <input type="radio" id="femenino" name="chgenero" value="femenino">
+    <label for="femenino">femenino</label>
+    <input type="radio" id="otro" name="chgenero" value="otro">
+    <label for="otro">otro</label>
+          </li>
+          <li>
+            <label for="color">color:</label>
+            <input type="color" name="color" id="color">
+          </li>
+          <li>
+            <label for="archivo">archivo:</label>
+            <input type="file" name="archivo" id="archivo">
+          </li>
+          <li>
+            <label for="sitio">sitio:</label>
+            <select name="pais">
+                <option value=" españa"><b>españa</b>  </option>
```

Comment:
For grouping options you must use [optgroup tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup)